src/pfe/portal/modules/User.js

```diff
@@ -102,6 +101,15 @@ module.exports = class User {
         log.error(`Codewind extensions failed to load. Error ${util.inspect(error)}`);
       }
 
+      // Initialise the list after we have created extensions as they may
+      // add template repositories. The log messages allow us to confirm
+      // how long this takes.
+      log.info(`Initialising template repository list`);
+      await this.templates.initializeRepositoryList();
+      log.info(`Initialising template list`);
+      await this.templates.getTemplates(false);
+      log.info(`Default templates cached.`);
```

Comment:
Massive nit: `Default templates cached.` has a full stop when the other `log`s don't.